### PR TITLE
CSS Optimization

### PR DESCRIPTION
**Pull request by chaitrali-r**

### CODE REVIEW

File: src/app/admin/create-vc-template/create-vc-template.component.html [hunk 1]
```diff
@@ -23,7 +24,7 @@
             <div class="thmbCol">
                 <div class="pt-c">
                   <a href="/add-template/{{usecase}}/{{entityName}}" class="btn submit-button pt-c">
-                     <i class="fa fa-plus text-secondary-color pt-c" aria-hidden="true"></i>
+                     <i class="fa fa-plus color-sea-green pt-c" aria-hidden="true"></i>
```
Comment (Pratikshakhandagale) [inline src/app/admin/create-vc-template/create-vc-template.component.html:27]: @chaitrali-r here you have rename the class name ? color-sea-green ?  is this secondary text color ?

File: src/app/admin/dlogin/dlogin.component.html [hunk 2]
```diff
@@ -11,49 +11,49 @@ <h3 class="fw-bold mb-4">{{'SCHEMA' | translate}}</h3>
 
         <div class="d-flex flex-row row mb-3">
             <div class="card p-4 col-sm-2 mr-2">
-                <p class="p16 mb-1 font-600">{{'FACILITY_CENTER' | translate}}</p>
+                <p class="fs-16 mb-1 font-600">{{'FACILITY_CENTER' | translate}}</p>
                 <h2 class="font-600">0</h2>
-                <p class="p12 mb-1">{{'RECORD' | translate}}</p>
-                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">{{'VIEW_ADD' | translate}}</a>
+                <p class="fs-12 mb-1">{{'RECORD' | translate}}</p>
+                <a class="my-2 fs-14 btn bg-color-light-blue color-sea-green submit-button align-self-start">{{'VIEW_ADD' | translate}}</a>
```
Comment (Pratikshakhandagale) [inline src/app/admin/dlogin/dlogin.component.html:17]: @chaitrali-r - bg-color-light-blue can't give name like this. we can change color code here. btn-sec-bg is correct one

File: src/app/admin/dlogin/dlogin.component.html [hunk 3]
```diff
@@ -11,49 +11,49 @@ <h3 class="fw-bold mb-4">{{'SCHEMA' | translate}}</h3>
...
             </div>
 
             <div class="card p-4 col-sm-2 mr-2">
-                <p class="p16 mb-1 font-600">{{'DOCTORS | translate'}}</p>
+                <p class="fs-16 mb-1 font-600">{{'DOCTORS | translate'}}</p>
                 <h2 class="font-600">0</h2>
-                <p class="p12 mb-1">{{'RECORD' | translate}}</p>
-                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">{{'VIEW_ADD' | translate}}</a>
+                <p class="fs-12 mb-1">{{'RECORD' | translate}}</p>
+                <a class="my-2 fs-14 btn bg-color-light-blue color-sea-green submit-button align-self-start">{{'VIEW_ADD' | translate}}</a>
```
Comment (Pratikshakhandagale) [inline src/app/admin/dlogin/dlogin.component.html:24]: @chaitrali-r dont give class name based on color code. Color code will be change for differnet theme.
- bg-color-light-blue 
- color-sea-green

File: src/app/admin/sidemenu/sidemenu.component.scss [hunk 4]
```diff
@@ -162,10 +162,10 @@ a.article:hover {
 
 //  Added css by - pratiksha
 
-.mtop6 {
-    margin-top: 3rem;
-}
+// .mtop6 {
+//     margin-top: 3rem;
```
Comment (Pratikshakhandagale) [inline src/app/admin/sidemenu/sidemenu.component.scss:166]: @chaitrali-r Remove this commented code

File: src/app/admin/create-vc-template/create-vc-template.component.html (hunk 1 above)
Comment (chaitrali-r) [reply src/app/admin/create-vc-template/create-vc-template.component.html:27]: Yes, i'll change the class name

File: src/app/admin/dlogin/dlogin.component.html (hunk 2 above)
Comment (chaitrali-r) [reply src/app/admin/dlogin/dlogin.component.html:17]: yes i had assigned a variable for color which changes according to theme, will change classname

**Pull request closed without merging**